docker/Dockerfile

```diff
@@ -1,25 +1,21 @@
-FROM rust:1.86.0-slim-bullseye AS builder
+FROM rust:1.87-alpine AS builder
```

Comment:
Please don't upgrade the Rust version. There are other places in the project where the Rust version is used.